interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -4372,14 +4372,16 @@ function start_your_engines($FIELDS)
             $term = $amihere['term'];
         }
 
-        if (stripos(($FIELDS[$amihere['location']] ?? ''), $term) !== false) {
+        $matches=[];
```

Comment:
FILE: /home/runner/work/openemr/openemr/interface/forms/eye_mag/php/eye_mag_functions.php
-----------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 2 LINES
-----------------------------------------------------------------------------------------
 4375 | ERROR | [x] Expected at least 1 space before "="; 0 found
 4375 | ERROR | [x] Expected at least 1 space after "="; 0 found
 4377 | ERROR | [x] Whitespace found at end of line
-----------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY